app/controllers/application_controller.rb

```diff
@@ -6,8 +6,4 @@ class ApplicationController < ActionController::Base
 
   responders :flash
   respond_to :html
-
-  decent_configuration do
```

Comment:
According to https://github.com/hashrocket/decent_exposure/wiki/Api-changes-in-version-3#strategies:

> If you have set in your ApplicationController to work with strong parameters you can just remove this code:
> ```ruby
# v 2.x
class ApplicationController < ActionController::Base
  decent_configuration do
    strategy DecentExposure::StrongParametersStrategy
  end
end
```

> And you should implement explicitly your #{exposed_thing}_params method like:
> ```ruby
# v 3.x
class ThingController < ApplicationController
  def thing_params
    params.require(:thing).permit(:foo, :bar)
  end
end
```